### PR TITLE
 updated history dependency

### PR DESCRIPTION
**Pull request by dendoesit**
**What?**

Updated history dependency to version 5 which doesn't support basename or hashtype.
We removed type declaration for history.

**Why?**

Monthly cleanup.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -39,12 +39,7 @@ const getStartPassage = () => {
   return data.startPassage
 }
 
-const history = createHashHistory({
-  basename: `/stories/${data.id}/${
-    window.location.href.includes('play') ? 'play' : 'test'
-  }`,
-  hashType: 'hashbang',
```
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply src/index.tsx:46]: I have no idea 😅

[the remark this answers is not in the document]
Comment (sampettersson) [reply src/index.tsx:46]: U need to point the story format to local and check that the play and test buttons works as they should in angel 👼

[the remark this answers is not in the document]
Comment (robinandeer) [reply src/index.tsx:46]: Okey, can do that! Any difference between "test" and "play"? Other than "test" starting from the middle of a story?

**Comment by dendoesit** [comment]
Picked this back up. 

Tested it on mine seems to work ok. 

https://user-images.githubusercontent.com/63097139/139686497-c38c22b5-4cc0-4bc7-a333-3b3beb49a688.mp4

**Review by sampettersson — APPROVED** [review]

**Pull request merged**